content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -105,29 +113,36 @@ You can view tracked metrics and plots with DVC. Let's start with the metrics:
 
 ```dvc
 $ dvc metrics show
-Path         avg_prec    roc_auc
-scores.json  0.52048     0.9032
+Path             avg_prec    roc_auc
+evaluation.json  0.89668     0.92729
 ```
 
 To view plots, first specify which arrays to use as the plot axes. We only need
 to do this once, and DVC will save our plot configurations.
 
 ```dvc
-$ dvc plots modify prc.json -x recall -y precision
+$ dvc plots modify evaluation/plots/precision_recall.json \
+                   -x recall -y precision
+Modifying stage 'evaluate' in 'dvc.yaml'
+$ dvc plots modify evaluation/plots/roc.json -x fpr -y tpr
 Modifying stage 'evaluate' in 'dvc.yaml'
-$ dvc plots modify roc.json -x fpr -y tpr
+$ dvc plots modify evaluation/plots/confusion_matrix.json \
+                   -x actual -y predicted -t confusion
 Modifying stage 'evaluate' in 'dvc.yaml'
+
 ```
 
 Now let's view the plots:
 
 ```dvc
 $ dvc plots show
-file:///Users/dvc/example-get-started/plots.html
+file:///Users/dvc/example-get-started/dvc_plots/index.html
 ```
 
 ![](/img/plots_prc_get_started_show.svg)
 ![](/img/plots_roc_get_started_show.svg)
+![](/img/plots_importance_get_started_show.png '=300 :wrap-left')
```

Comment:
This was also commented in other P.Rs but font sizes are kind of small when we add this kind of images. Not sure how critical it is though, as users can click to expand

Reply:
yep, I think it's more or less fine and we can improve this later